nuitka/plugins/standard/PkgResourcesPlugin.py

```diff
@@ -48,11 +48,11 @@ def __init__(self):
             self.metadata = importlib_metadata
 
         try:
-            import metadata
-
-            self.metadata = metadata
+            from importlib import metadata
         except ImportError:
-            pass
+            self.metadata = None
```

Comment:
That just breaks older Python with `importlib_metadata` but not `metadata` does it not? I cannot see how it would fix the issue you are having. The existing code takes care to write `self.metadata` only if it looks like something that ought to be preferred.

Reply:
Can you revert this part though. It's not needed for your think. I think I can merge it then. Thanks a lot!